README.md

```diff
@@ -84,7 +84,7 @@ simply close your current terminal, open a new terminal, and try verifying again
 
 **Note:** On OS X, if you get `nvm: command not found` after running the install script, one of the following might be the reason:-
 
-  - your system may not have a [`.bash_profile file`] where the command is set up. Simply create one with `touch ~/.bash_profile` and run the install script again
+  - your system may not have a [`.bash_profile` file] where the command is set up. Simply create one with `touch ~/.bash_profile` and run the install script again
```

Comment:
actually i'm confused where this is supposed to link to; git blame suggests the brackets are an error. (also, i removed "simply")
```suggestion
  - your system may not have a `.bash_profile` file where the command is set up. Create one with `touch ~/.bash_profile` and run the install script again
```